rootfs/runtimes/nodejs/workload.sh

```diff
@@ -1,7 +1,2 @@
 #!/usr/bin/env sh
-
-/usr/bin/setup-eth0.sh
```

Comment:
Why remove the network setup? This, I think, is necessary unless it's already happening elsewhere

Reply:
This gives me the following error message
```
ip: ioctl 0x8913 failed: No such device
ip: can't find device 'eth0'
ip: can't find device 'eth0'
```
so I thought it was irrelevant and removed it. Also, I tried to fix it but couldn't find any fix. Do you have an idea what's going on? Maybe I can look into this.

Reply:
Also ip link only gives me `lo`

Reply:
Ok, I add it back. It still leaves me with this error but since it is not fatal, I'll configure the network device at some point in the future to get rid of it.